week-2-practise/level-3/3-sort.js

```diff
@@ -4,7 +4,11 @@
 
 var nums = [10, 1, 5, 29, 100];
 var nums2 = [11, 6, 3, 29, 12];
-var sortedNums; // complete this statement
+var allNums = nums.concat(nums2);
+function newNums(one, two) {
```

Comment:
What does this function do (read up on the `sort` function if you aren't sure)? Is `newNums` a good name for it?